tgui/packages/tgui/interfaces/Secrets.js

```diff
@@ -0,0 +1,723 @@
+        <Flex.Item grow={1}>
+          <Button
+            align="center"
+            icon="pencil-alt"
+            lineHeight={lineHeightNormal}
+            fluid
+            content="Rename Station"
+            onClick={() => act("set_name")} />
+        </Flex.Item>
+        <Flex.Item grow={1} ml={0.5}>
+          <Button
+            align="center"
+            icon="eraser"
+            lineHeight={lineHeightNormal}
+            fluid
+            content="Reset Station"
+            onClick={() => act("reset_name")} />
+        </Flex.Item>
+      </Flex>
+      <Flex
+        grow={1}
+        direction="row"
+        height="100%"
+        align="stretch"
+        justify="space-between">
+        <Flex.Item grow={1}>
+          <Button
+            align="center"
+            icon="plane-departure"
+            lineHeight={lineHeightNormal}
+            fluid
+            content="Move Ferry"
+            onClick={() => act("moveferry")} />
+        </Flex.Item>
+        <Flex.Item grow={1} mx={0.5}>
+          <Button
+            align="center"
+            icon="plane"
+            lineHeight={lineHeightNormal}
+            fluid
+            content="Toggle Arrivals"
+            onClick={() => act("togglearrivals")} />
+        </Flex.Item>
+        <Flex.Item grow={1}>
+          <Button
+            align="center"
+            icon="plane-arrival"
+            lineHeight={lineHeightNormal}
+            fluid
+            content="Move Labor"
+            onClick={() => act("movelaborshuttle")} />
+        </Flex.Item>
+      </Flex>
+    </Flex>
+  );
+};
+
+const FunTab = (props, context) => {
+  const { act } = useBackend(context);
+  return (
+    <Flex direction="column" mb={-0.75} mx={-0.5}>
+      <Flex
+        mb={1}
+        grow={1}
+        direction="row"
+        height="100%"
+        align="stretch"
+        justify="space-between">
+        <Flex.Item grow={1}>
+          <Button
+            align="center"
+            icon="grin-beam-sweat"
+            lineHeight={lineHeightNormal}
+            fluid
+            content="Break all lights"
+            onClick={() => act("blackout")} />
+        </Flex.Item>
+        <Flex.Item grow={1} mx={0.5}>
+          <Button
+            align="center"
+            icon="magic"
+            lineHeight={lineHeightNormal}
+            fluid
+            content="Fix all lights"
+            onClick={() => act("whiteout")} />
+        </Flex.Item>
+        <Flex.Item grow={1}>
+          <Button
+            align="center"
+            icon="biohazard"
+            lineHeight={lineHeightNormal}
+            fluid
+            content="Trigger Outbreak"
+            onClick={() => act("virus")} />
+        </Flex.Item>
+      </Flex>
+      <Flex
+        mb={1}
+        grow={1}
+        direction="row"
+        height="100%"
+        align="stretch"
+        justify="space-between">
+        <Flex.Item>
+          <Button
+            align="center"
```

Comment:
Align does nothing on Button, if you want to align all text to center, just put textAlign on the most upper level element, and all children will inherit it.